apps/frontend/src/views/Results.vue

```diff
@@ -218,7 +218,7 @@ import ExportXCCDFResults from '@/components/global/ExportXCCDFResults.vue';
 import RouteMixin from '@/mixins/RouteMixin';
```

Comment:
the snackbar notification that appears after you deselect your last file should mention that you could also load in files from the load button, not just say to select a file

Reply:
Added "If you would like to load a file, press the `<LOAD-icon>` button above. "

Reply:
Also removed the side effect in `canClear` which controlled the snackbar